gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java

```diff
@@ -31,7 +32,7 @@
   private static final long serialVersionUID = 5640180400046623305L;
   private static final String PUBSUB_SCOPE = "https://www.googleapis.com/auth/pubsub";
   private static final Set<String> SCOPES = ImmutableSet.of(PUBSUB_SCOPE);
-  private static final String DEFAULT_HOST = "pubsub.googleapis.com";
+  private static final String DEFAULT_HOST = PublisherSettings.getDefaultServiceAddress();
```

Comment:
This could be done also for `LoggingOptions`. But I can also do that in a separate PR where I also refactor host-setting code for both services.